modules/nodes-pods-autoscaling-creating-web-console.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+[id="nodes-pods-autoscaling-creating-web-console_{context}"]
+
+= Creating a horizontal pod autoscaler using web console
+
+You can create a horizontal pod autoscaler to specify the minimum and maximum number of pods you want to run on a standalone deployment or deployment configurations. You can define the CPU utilization or memory utilization your pods should target.
+
+[NOTE]
+====
+Horizontal pod autoscaler cannot be added to those Deployment or Deployment Configs that are part of Operator Backed Service, Knative Service, or Helm Chart.
+====
+
+.Procedure
+
+You can create, edit, and remove a horizontal pod autoscaler in the web console.
+
+* Create a horizontal pod autoscaler:
+** In the *Topology* view, click the node to reveal the side pane.
+** Click *Actions* drop down or right click the pod and select the *Add HorizontalPodAutoscaler*, that will open the *Add HorizontalPodAutoscaler* form page.
```

Comment:
Something like this:
```suggestion
** In the *Topology* view, right click the pod and select the *Add HorizontalPodAutoscaler* to open the *Add HorizontalPodAutoscaler* form page.
** Or click the node to reveal the side pane and click *Actions* drop down and select the *Add HorizontalPodAutoscaler*, that will open the *Add HorizontalPodAutoscaler* form page.
```

Reply:
@sanketpathak  Can we keep only one option to perform this task? Only from the side panel and design document also implies most of the steps from there only

Reply:
I think we can keep the side panel option only and may be in a note can include that right click provides that option as well or we can leave it completely as options in the action menu are always found in the context menu of the pod(right click)